packages/components/src/TextAreaField/_docs/TextAreaField.mdx

```diff
@@ -0,0 +1,47 @@
+import { Canvas, Controls, Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as TextAreaFieldStories from "./TextAreaField.stories"
+
+<Meta of={TextAreaFieldStories} />
+
+# TextAreaField
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/TextAreaField"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=891%3A179030&t=zlNKMVZd1i6MB4db-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3081895966/Text+Area+Field"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { TextAreaField } from "@kaizen/components"'
+/>
+
+## Overview
+
+A text area includes a label and a longer area you can type multiple lines of text into.
```

Comment:
Could do the same thing here that you did with TextField mentioning description and validation